prisma/migrations/20241128151318_created_link_statistics_table/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- CreateTable
+CREATE TABLE `LinkStatistic` (
+    `id` VARCHAR(191) NOT NULL,
```

Comment:
**suggestion**: Consider using a UUID for the `id` field to ensure global uniqueness.